torch/_torch_docs.py

```diff
@@ -2845,6 +2845,34 @@ def merge_dicts(*dicts):
         tensor([False,  True,  False,  True,  False])
 """)
 
+add_docstr(torch.isposinf,
+           r"""
+isposinf(input, *, out=None) -> Tensor
+Tests if each element of :attr:`input` is positive infinity or not.
+Args:
+    {out}
+Example::
+    >>> a = torch.tensor([-float('inf'), float('inf'), 1.2])
+    >>> a
```

Comment:
No need to print `a` after you've shown how it's constructed.

Reply:
addressed.